CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## 3.33.0 (Unreleased)
+* resource/cloudflare_teams_accounts: adds support for mailto blockpage settings ([#2132](https://github.com/cloudflare/terraform-provider-cloudflare/issues/2132))
```

Comment:
check out https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/changelog-process.md which tells you how to have this line automatically generated :)

Reply:
fixed!